### PR TITLE
Add check command

### PR DESCRIPTION
**Pull request by ohad83**
Solve #1 

Add a check command to check whether the currently generated files are up to date.

### CODE REVIEW

File: bin/index.js [hunk 1]
```diff
@@ -66,6 +67,24 @@ const convertCommand = {
...
+  desc: "Local path of exported browser bookmarks",
+  handler: async (argv) => {
+    console.log("CHECKING BOOKMARKS");
+    const checkResult = await checkBookmarks.checkBookmarks(
+      argv.bookmarks,
+      argv.directory
+    );
+    if (!checkResult) {
+      console.log("Bookmarks are not up to date");
+      process.exitCode = 1;
```
Comment (robinglen) [inline bin/index.js]: for consistency the rest or the project uses `process.exit(1)`

File: docs/ADVANCED-USAGE.md [hunk 2]
```diff
@@ -63,3 +63,16 @@ $ npx convert ./browsers.html -f="./my-bookmarks.yaml"
 $ ls
 my-bookmarks.yaml
 ```
+
+## Check
+
+Check whether current generated files are up to date with the yaml file.
```
Comment (robinglen) [inline docs/ADVANCED-USAGE.md:69]: I wonder if its worth documenting here why you might want to use this?

File: src/check-bookmarks.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  try {
+    existingBookmark = readFileSync(filename, 'utf8');
+  } catch (e) {
+    console.log(`Failed to read file ${filename}`);
+    return false;
+  }
+  return checkBookmarkBody(existingBookmark, generatedBookmark.body);
+}
+
+export { checkBookmarks, checkSingleBookmark as checkBookmark, sanitizeDynamicData, checkBookmarkBody };
```
Comment (robinglen) [inline src/check-bookmarks.js]: I'm wondering why export as different name and why not just use that name for the function?

File: src/index.js [hunk 4]
```diff
@@ -3,11 +3,13 @@ import * as generateBookmarks from "./generate-bookmarks.js";
 import * as saveBookmarks from "./save-bookmarks.js";
 import * as mergeBookmarks from "./merge-bookmarks.js";
 import * as convertBookmarks from "./convert-bookmarks.js";
+import * as checkBookmarks from "./check-bookmarks.js";
```
Comment (robinglen) [inline src/index.js:6]: I think we should also add this to the developer readme my to explain how the functions can be used to be consistent with other modules. Its just a quick note for people who might want to import into other projects

**Review by robinglen — COMMENTED** [review]
Here are some comments, all small mainly for consistency but it looks good!

File: docs/ADVANCED-USAGE.md (hunk 2 above)
Comment (ohad83) [reply docs/ADVANCED-USAGE.md:69]: Added some documentation, hope that's what you meant.

File: src/check-bookmarks.js (hunk 3 above)
Comment (ohad83) [reply src/check-bookmarks.js]: Just an IDE brainfart, I used a rename feature of the IDE and it did that. Fixed it to use the same name.

File: src/index.js (hunk 4 above)
Comment (ohad83) [reply src/index.js:6]: Added documentation.

**Comment by robinglen** [comment]
Looks good, thanks so much for the contribution I will merge and publish

**Pull request merged**